### ISSUE TITLE
ABR: NoSuchElementException

### ISSUE THREAD

**Issue by uckelman** [vassalengine/vassal#6928]
| Bugzilla |  |
|----------|----------|
| ID | BZ9790 |
| Reported | 2013-01-29 10:14:00 +0100 |
| Modified | 2013-02-07 13:43:01 +0100 |
| Product | VASSAL |
| Component | Unknown |
| Version | 3.2.2 |
| Platform | All |
| OS | All |
| Status | RESOLVED |
| Resolution | DUPLICATE |
| Priority | unspecified |
| Severity | normal |
| Milestone | --- |
| Reporter | [Joel Uckelman](mailto:uckelman@nomic.net) |
| Assignee | [Joel Uckelman](mailto:uckelman@nomic.net) |
| CC | ainulindale@gmail.com |
| Duplicate of | BZ2569 |

| Attachments | |
|-------|-------|
| [errorLog](http://www.vassalengine.org/files/bz/9010) | [Joel Uckelman](mailto:uckelman@nomic.net) 2013-01-29 10:14:00 +0100 text/plain 172828 the errorLog |

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2013-01-29 10:14:36 +0100
<pre>---- Reported by uckelman@nomic.net  ----

Reported by waiting on typing

--- Bug imported by uckelman@nomic.net 2013-01-29 10:14 MST  ---

This bug was previously known as _bug_  at http://www.vassalengine.org/trackershow_bug.cgi?id=
Imported an attachment (id=9010)

The original submitter of attachment 9010 is unknown.
   Reassigning to the person who moved it here: uckelman@nomic.net.</pre>

#### [Julien Cassignol](mailto:ainulindale@gmail.com) 2013-02-07 13:43:01 +0100
<pre>*** This bug has been marked as a duplicate of bug 2569 ***</pre>

**Comment by uckelman** [comment]
Duplicate of #5059
